Bojzen-I-Mitten/Stort-spelprojekt
# Menu/select team

New selectTeam menu and two new 3D-backgrounds.
One for MainMenu and one for HostGame and SelectTeam.

To test:
- Update SVN and use the DemoSceneSelectTeam.tds
- Start the game and you will see the new main menu background.
- Play
- Host
-You will now see the second new 3D-background.
- Host
- Hover over one team/chad. (He will start to run)
- Done!
